launchpad/index.d.ts

```diff
@@ -0,0 +1,108 @@
+// Type definitions for launchpad
+// Project: https://www.npmjs.com/package/launchpad
+// Definitions by: Peter Burns <https://github.com/rictic>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare module "launchpad" {
```

Comment:
do not use external ambient module declarations. just write it as a normal ES6 module, with top level exports. this avoids polluting the global namespace and conflicts down the road if two versions of the module were loaded.